includes/Modules/Idea_Hub.php

```diff
@@ -348,9 +353,14 @@ private function admin_notice_idea_hub_ideas( $notices ) {
 			array(
 				'content'         => function() use ( $escape_and_wrap_notice_content ) {
 					$message = sprintf(
-						/* translators: %s: URL to new ideas */
-						__( 'Site Kit by Google: Want some inspiration for a new post? <a href="%s">Review your new ideas</a> in Site Kit.', 'google-site-kit' ),
-						esc_url( $this->context->admin_url( 'dashboard', array( 'idea-hub-tab' => 'new-ideas' ) ) )
+						/* translators: %1$s: Plugin name. %2$s: Notice content. */
+						__( '%1$s: %2$s', 'google-site-kit' ),
+						get_plugin_data( GOOGLESITEKIT_PLUGIN_MAIN_FILE )['Name'],
+						sprintf(
```

Comment:
Nested `sprintf` calls here to keep the original translated notice text as it was. Feedback is very welcome if this should be addressed with a different approach.

Reply:
See above

Reply:
Sorted.